doc/cephadm/index.rst

```diff
@@ -1,11 +1,16 @@
-.. _bootstrap:
+.. _cephadm-bootstrap:
 
 ========================
  Installation (cephadm)
 ========================
 
 .. note:: The *cephadm* bootstrap feature is first introduced in Octopus, and is not yet recommended for production deployments.
 
+cephadm is an orchestrator that does not rely
+on a separate system such as Rook or Ansible, but rather manages nodes
+in a cluster by establishing an SSH connection and issuing explicit
+management commands.
```

Comment:
I'd consider rephrasing this so the fact is first. IE:

"cephadm manages nodes in a cluster by establishing an SSH connection and issues explicit management commands. It doesnot rely on separate systems such as Rook or Ansible."

Reply:
fixed